GOVERNANCE.md

```diff
@@ -63,10 +63,9 @@ A guide for Collaborators is maintained in
 
 ## CTC Membership
 
-CTC seats are not time-limited.  There is no fixed size of the CTC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
+CTC seats are not time-limited. There is no fixed size of the CTC. The CTC
+should be of such a size as to ensure adequate coverage of important areas of
```

Comment:
If we want to add diversity as a consideration, I support that. But it was not in the original text and I think it should go in as a separate PR. This PR is for getting rid of the obsolete target size.